modules/internal/Bytes.chpl

```diff
@@ -21,7 +21,7 @@
 /*
 The following document shows functions and methods used to
 manipulate and process Chapel bytes objects. The :record:`bytes` object is
-similar to astringbut allows arbitrary data to be stored in it. Methods onbytes`
+similar to a :record:`string` but allows arbitrary data to be stored in it. Methods on :record:`bytes`
```

Comment:
I'm not completely sure these should both be tagged with `:record:`, but this is my best guess.  I left the line unwrapped to make it easier to compare and fix up, but I can certainly reflow it if desired.